packages/katello/katello/backup.rb

```diff
@@ -260,10 +260,14 @@ def online_backup(database)
           puts "Backing up Pulp data... "
           matching = false
           until matching
+            FileUtils.cp File.join(@dir, '.pulp.snar'), File.join(@dir, '.pulp.snar.backup') if File.exist? File.join(@dir, '.pulp.snar')
             checksum1 = run_cmd("find . -printf '%T@\n' | md5sum")
             create_pulp_data_tar
             checksum2 = run_cmd("find . -printf '%T@\n' | md5sum")
             matching = (checksum1 == checksum2)
+            if File.exist? File.join(@dir, '.pulp.snar.backup')
+              matching ? FileUtils.rm(File.join(@dir, '.pulp.snar.backup')) : FileUtils.mv(File.join(@dir, '.pulp.snar.backup'), File.join(@dir, '.pulp.snar'))
```

Comment:
considering the length of the line, my personal preference would be to break this up to if/else block instead of a ternary operator. It's not a blocker considering we don't run rubocop and have many similar cases in this file, so its up to you :)

Reply:
I thought about that, but given the extended length of many other lines of this particular file, felt that we should keep it more uniform with our general code (using ternary operator vs if/else for single lines). Let me know if you'd still like me to break it up, and I will do so.

Reply:
After more thought, I came up with a solution that will clean up the code a bit as well.